src/components/lyne-link-list/lyne-link-list.custom.d.ts

```diff
@@ -1,3 +1,4 @@
 export interface InterfaceLyneLinkListAttributes {
   variant?: 'positive' | 'negative';
```

Comment:
To follow your recommendation, we could change **variant** to **appearance**. Then we could still decide whether vertical | horizontal are better named variant or direction. Direction is very specific and makes great sense for the current case while variant could offer more options for the future.

Reply:
This would definitely be an option to rename the **variant** into **appearance** but this would also require to rename the `lyne-title` since the variant gets passed into `lyne-title`. I won't do the refactoring within this branch but we can definitely consider to refactor these components. 

WIth merging the direction property into the variant/appearance property I do not fully agree. Because if we would do so, we would loose the opportunity to combine the variant with the direction like we can at the moment. We would have to create four variants ... 

- positive-vertical
- positive-horizontal
- negative-vertical
- negative-horizontal